java/src/jmri/jmrix/openlcb/OlcbClockControl.java

```diff
@@ -126,7 +126,10 @@ public void startHardwareClock(Date now) {
 
     @Override
     public void setRate(double newRate) {
-        hardwareClock.requestSetRate(newRate);
+        // OpenLCB rates are 0.25 resolution, so we use half of that as minimum threshold.
```

Comment:
Not sure what this test is trying to do.  The internal clock rate can be adjusted pretty finely, and people sometimes do that to e.g. match the rate of an external (but non-connected) clock.  One local layout runs at 2.95 for example. It might be confusing to have some entries propagate and some not. If the user tries to set something not exactly available, does the updated (i.e. "I can only do 3.0, not 2.95") propagate back to the clocks?